pcf/test/particle/aws/test_particles.py

```diff
@@ -0,0 +1,54 @@
+import moto
+import json
+import pytest
+import boto3
+import os
+import placebo
+from pcf.core import State
+from pcf.core import particle_flavor_scanner
+from contextlib import ExitStack
+
+from pcf.particle.aws.route53 import hosted_zone
+from pcf.particle.aws.sqs.sqs_queue import SQSQueue
+from pcf.particle.aws.cloudfront.cloudfront import CloudFront
```

Comment:
we will switch to use jacobs dynamic flavor importer so we dont need to add these

Reply:
do i just get rid of them, or do i need to set something up?

Reply:
we need to wait for his pr to get merged so just fyi